pkgs/applications/video/qmplay2/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv
+, fetchFromGitHub
+, pkg-config
+, cmake
+, alsaLib
+, ffmpeg
+, libass
+, libcddb
+, libcdio
+, libgme
+, libpulseaudio
+, libsidplayfp
+, libva
+, libXv
+, taglib
+, qtbase
+, qttools
+, wrapQtAppsHook
+}:
+
+let
+  pname = "qmplay2";
+  version = "20.07.04";
+in stdenv.mkDerivation {
+  inherit pname version;
```

Comment:
```suggestion
stdenv.mkDerivation rec {
  pname = "qmplay2";
  version = "20.07.04";
```

Reply:
Why? As the people said, it reduces the scope. No need for using `rec` here.

https://nix.dev/anti-patterns/index.html

Reply:
See https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293